From khali@linux-fr.org Mon Feb 27 14:13:02 2006
Date: Mon, 27 Feb 2006 23:13:53 +0100
From: Jean Delvare <khali@linux-fr.org>
To: Greg KH <greg@kroah.com>
Subject: I2C: Drop unneeded i2c-dev.h includes
Message-Id: <20060227231353.9e7f6cc9.khali@linux-fr.org>
Content-Disposition: inline; filename=i2c-drop-unneeded-i2c-dev-h-includes.patch

Several media/video and sound drivers include i2c-dev.h while they
don't need it at all. Clean it up.

This header file is really only needed by i2c-dev.c and
compat_ioctl.c, other drivers should never need it.

Signed-off-by: Jean Delvare <khali@linux-fr.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/media/video/adv7170.c       |    1 -
 drivers/media/video/adv7175.c       |    1 -
 drivers/media/video/bt819.c         |    1 -
 drivers/media/video/bt856.c         |    1 -
 drivers/media/video/saa7110.c       |    1 -
 drivers/media/video/saa7111.c       |    1 -
 drivers/media/video/saa7114.c       |    1 -
 drivers/media/video/saa711x.c       |    1 -
 drivers/media/video/saa7185.c       |    1 -
 drivers/media/video/vpx3220.c       |    1 -
 sound/oss/dmasound/dmasound_awacs.c |    2 --
 sound/ppc/daca.c                    |    1 -
 sound/ppc/keywest.c                 |    1 -
 sound/ppc/toonie.c                  |    1 -
 sound/ppc/tumbler.c                 |    1 -
 15 files changed, 16 deletions(-)

--- gregkh-2.6.orig/drivers/media/video/adv7170.c
+++ gregkh-2.6/drivers/media/video/adv7170.c
@@ -53,7 +53,6 @@ MODULE_AUTHOR("Maxim Yevtyushkin");
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(x) (x)->name
 
--- gregkh-2.6.orig/drivers/media/video/adv7175.c
+++ gregkh-2.6/drivers/media/video/adv7175.c
@@ -49,7 +49,6 @@ MODULE_AUTHOR("Dave Perks");
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(s) (s)->name
 
--- gregkh-2.6.orig/drivers/media/video/bt819.c
+++ gregkh-2.6/drivers/media/video/bt819.c
@@ -53,7 +53,6 @@ MODULE_AUTHOR("Mike Bernson & Dave Perks
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(s) (s)->name
 
--- gregkh-2.6.orig/drivers/media/video/bt856.c
+++ gregkh-2.6/drivers/media/video/bt856.c
@@ -53,7 +53,6 @@ MODULE_AUTHOR("Mike Bernson & Dave Perks
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(s) (s)->name
 
--- gregkh-2.6.orig/drivers/media/video/saa7110.c
+++ gregkh-2.6/drivers/media/video/saa7110.c
@@ -39,7 +39,6 @@ MODULE_AUTHOR("Pauline Middelink");
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(s) (s)->name
 
--- gregkh-2.6.orig/drivers/media/video/saa7111.c
+++ gregkh-2.6/drivers/media/video/saa7111.c
@@ -52,7 +52,6 @@ MODULE_AUTHOR("Dave Perks");
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(s) (s)->name
 
--- gregkh-2.6.orig/drivers/media/video/saa7114.c
+++ gregkh-2.6/drivers/media/video/saa7114.c
@@ -55,7 +55,6 @@ MODULE_AUTHOR("Maxim Yevtyushkin");
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(x) (x)->name
 
--- gregkh-2.6.orig/drivers/media/video/saa711x.c
+++ gregkh-2.6/drivers/media/video/saa711x.c
@@ -45,7 +45,6 @@ MODULE_AUTHOR("Dave Perks, Jose Ignacio 
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(s) (s)->name
 
--- gregkh-2.6.orig/drivers/media/video/saa7185.c
+++ gregkh-2.6/drivers/media/video/saa7185.c
@@ -49,7 +49,6 @@ MODULE_AUTHOR("Dave Perks");
 MODULE_LICENSE("GPL");
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(s) (s)->name
 
--- gregkh-2.6.orig/drivers/media/video/vpx3220.c
+++ gregkh-2.6/drivers/media/video/vpx3220.c
@@ -30,7 +30,6 @@
 #include <asm/uaccess.h>
 
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 
 #define I2C_NAME(x) (x)->name
 
--- gregkh-2.6.orig/sound/oss/dmasound/dmasound_awacs.c
+++ gregkh-2.6/sound/oss/dmasound/dmasound_awacs.c
@@ -88,8 +88,6 @@
 #include <linux/pmu.h>
 #endif
 
-#include <linux/i2c-dev.h>
-
 #include <asm/uaccess.h>
 #include <asm/prom.h>
 #include <asm/machdep.h>
--- gregkh-2.6.orig/sound/ppc/daca.c
+++ gregkh-2.6/sound/ppc/daca.c
@@ -22,7 +22,6 @@
 #include <sound/driver.h>
 #include <linux/init.h>
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 #include <linux/kmod.h>
 #include <linux/slab.h>
 #include <sound/core.h>
--- gregkh-2.6.orig/sound/ppc/keywest.c
+++ gregkh-2.6/sound/ppc/keywest.c
@@ -23,7 +23,6 @@
 #include <linux/init.h>
 #include <linux/i2c.h>
 #include <linux/delay.h>
-#include <linux/i2c-dev.h>
 #include <linux/slab.h>
 #include <sound/core.h>
 #include "pmac.h"
--- gregkh-2.6.orig/sound/ppc/toonie.c
+++ gregkh-2.6/sound/ppc/toonie.c
@@ -22,7 +22,6 @@
 #include <linux/init.h>
 #include <linux/delay.h>
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 #include <linux/kmod.h>
 #include <linux/slab.h>
 #include <linux/interrupt.h>
--- gregkh-2.6.orig/sound/ppc/tumbler.c
+++ gregkh-2.6/sound/ppc/tumbler.c
@@ -28,7 +28,6 @@
 #include <linux/init.h>
 #include <linux/delay.h>
 #include <linux/i2c.h>
-#include <linux/i2c-dev.h>
 #include <linux/kmod.h>
 #include <linux/slab.h>
 #include <linux/interrupt.h>
